docs/fr/docs/advanced/response-directly.md

```diff
@@ -1,63 +1,63 @@
 
-Then, behind the scenes, it would put that JSON-compatible data (e.g. a `dict`) inside of a `JSONResponse` that would be used to send the response to the client.
+Ensuite, dans les coulisses, il mettra ces données compatibles JSON (par exemple un `dict`) à l'intérieur d'un `JSONResponse` qui sera utilisé pour envoyer la réponse au client.
 
-But you can return a `JSONResponse` directly from your *path operations*.
+Mais vous pouvez retourner une `JSONResponse` directement à partir de vos *opérations de chemin*.
 
-It might be useful, for example, to return custom headers or cookies.
+Cela peut être utile, par exemple, pour retourner des en-têtes personnalisés ou des cookies.
 
-## Return a `Response`
+## Retourner une `Response`
 
-In fact, you can return any `Response` or any sub-class of it.
+En fait, vous pouvez retourner n'importe quelle `Response` ou n'importe quelle sous-classe de celle-ci.
 
-!!! tip
-    `JSONResponse` itself is a sub-class of `Response`.
+!!! Note
+    `JSONResponse` est lui-même une sous-classe de `Response`.
 
-And when you return a `Response`, **FastAPI** will pass it directly.
+Et quand vous retournez une `Response`, **FastAPI** la transmet directement.
 
-It won't do any data conversion with Pydantic models, it won't convert the contents to any type, etc.
+Elle ne fera aucune conversion de données avec les modèles Pydantic, elle ne convertira pas le contenu en un type quelconque.
 
-This gives you a lot of flexibility. You can return any data type, override any data declaration or validation, etc.
+Cela vous donne beaucoup de flexibilité. Vous pouvez retourner n'importe quel type de données, surcharger n'importe quelle déclaration ou validation de données.
 
-## Using the `jsonable_encoder` in a `Response`
+## Utiliser le `jsonable_encoder` dans une `Response`
 
-Because **FastAPI** doesn't do any change to a `Response` you return, you have to make sure it's contents are ready for it.
+Parce que **FastAPI** n'apporte aucune modification à une `Response` que vous retournez, vous devez vous assurer que son contenu est prêt à être utilisé.
 
-For example, you cannot put a Pydantic model in a `JSONResponse` without first converting it to a `dict` with all the data types (like `datetime`, `UUID`, etc) converted to JSON-compatible types.
+Par exemple, vous ne pouvez pas mettre un modèle Pydantic dans une `JSONResponse` sans d'abord le convertir en un `dict` avec tous les types de données (comme `datetime`, `UUID`, etc) convertis en types compatibles avec JSON.
 
-For those cases, you can use the `jsonable_encoder` to convert your data before passing it to a response:
+Pour ces cas, vous pouvez spécifier un appel à `jsonable_encoder` pour convertir vos données avant de les passer à une réponse :
 
-```Python hl_lines="6-7  21-22"
-{!../../../docs_src/response_directly/tutorial001.py!}
+``Python hl_lines="6-7 21-22"
+{ !../../../docs_src/response_directly/tutorial001.py!}
 ```
```

Comment:
```
```Python hl_lines="6-7  21-22"
{!../../../docs_src/response_directly/tutorial001.py!}
```

There is a missing ` before the word python. 
There is also a lack or too much whitespace.

Reply:
I correct this, if it's ok for you, please, close the comment. Thanks.